src/v/pandaproxy/types.h

```diff
@@ -52,4 +52,9 @@ struct credential_t {
       , pass{std::move(p)} {}
 };
 
+struct always_true : public config::binding<bool> {
+    always_true()
+      : config::binding<bool>(true) {}
+};
```

Comment:
nit: Now that this is more public, and has a wider scope, perhaps a name like `pp::config::always_true` might be more suitable?